episodes/03-create.md

```diff
@@ -161,30 +147,22 @@ his last `git init` in the `desserts` subdirectory?
 
 Removing files from a Git repository needs to be done with caution. But we have not learned
 yet how to tell Git to track a particular file; we will learn this in the next episode. Files
-that are not tracked by Git can easily be removed like any other "ordinary" files with
+that are not tracked by Git can easily be removed like any other "ordinary" files by deleting them in VSCode Explorer.
 
-```bash
-$ rm filename
-```
-
-Similarly a directory can be removed using `rm -r dirname`.
+Similarly a directory can be removed in the same way in Explorer.
 If the files or folder being removed in this fashion are tracked by Git, then their removal
 becomes another change that we will need to track, as we will see in the next episode.
 
 ### Solution
 
 Git keeps all of its files in the `.git` directory.
 To recover from this little mistake, Alfredo can remove the `.git`
-folder in the desserts subdirectory by running the following command from inside the `recipes` directory:
-
-```bash
-$ rm -rf desserts/.git
-```
+folder in the desserts subdirectory by deleting the `.git` directory.
 
 But be careful! Running this command in the wrong directory will remove
 the entire Git history of a project you might want to keep.
-In general, deleting files and directories using `rm` from the command line cannot be reversed.
-Therefore, always check your current directory using the command `pwd`.
+Further, although the deleted files and directories might be in the Recycle bin which can be recovered, we should not rely on this. Also the "Undo" command in VSCode does not always work to revert a delete command.
```

Comment:
```suggestion
Further, although the deleted files and directories might be in the Recycle bin which can be recovered, 
we should not rely on this. 
Also the "Undo" command in VSCode does not always work to revert a deleted file or directory.
```
Into multiple lines. Also, does VSCode "sometimes" work to revert deleted files?